Tools/scripts/pathfix.py

```diff
@@ -64,6 +67,11 @@ def main():
             create_backup = False
         if o == '-k':
             keep_flags = True
+        if o == '-a':
+            add_flags = a.encode()
+            if ' ' in a:
+                err('Does not support arguments')
```

Comment:
This error message is vague. I suggest:

```suggestion
                err("-a option doesn't support whitespaces")
```